### PR TITLE
EOS-15453: Update documentation related to EOS-14771 and EOS-15064

### PR DESCRIPTION
**Pull request by pratyush-seagate**
#  EOS-15453: Update documentation related to EOS-14771 and EOS-15064

## Checklist
- [x] **Compilation:** _This patch does not break compilation_
- [x] **Merge conflicts:** _This patch has been squashed and re-based, it can be merged using fast-forward merge_
- [] **Code review:** _All discussions have been resolved_
- [x] **Sanity Testing:** _All Unit tests are passing and able to do mount and io operations works from NFS client_
- [x] **Documentation:** _This patch and merge request have up to date description_
- [x] **Unit Testing and debugging:** _Both single and multi node changes are unit tested_

## UT
Not required as there is no source code change.

## Commit Message
commit 75e21083978f295ed2f5d198bd1cad46f5a1c969
Author: pratyush-seagate <pratyush.k.khan@seagate.com>
Date:   Wed Dec 2 12:16:14 2020 +0000

        EOS-15453: Update documentation related to EOS-14771 and EOS-15064
        List of modified files:
                modified:   doc/Quick_Start_Guide.md
                modified:   doc/Test_Instructions.md
        Change description:
                EOS-14771 introduces new build script configurable flag ENABLE_TSDB_ADDB to enable optional ADDB integrated performance profiling feature in cortxfs stack.
        EOS-15064 adds a new parameter fs block size to cortxfscli fs create command.
        The documentation is  updated for these two changes.

    Signed-off-by: pratyush-seagate <pratyush.k.khan@seagate.com>

### CODE REVIEW

File: doc/Test_Instructions.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ optional arguments:
   -h, --help          show this help message and exit
 ```  
 
-* Run the below mentioned command to create FS.
+* Run the below mentioned command to create FS. Valid block sizes are 2^12 to 2^20.
 
-   `cortxfscli fs create <fs_name>`
+   `cortxfscli fs create <fs_name> proto=nfs,fs_bsize=<block_size>`
```
Comment (sachinpunadikar) [inline doc/Test_Instructions.md:46]: Is proto required here ?

Comment (pratyush-seagate) [reply doc/Test_Instructions.md:46]: Yes the proto is required. The block size parameter is currently set as mandatory parameter for cortxfscli tool, but it is an optional parameter for the management interface.

Comment (sachinpunadikar) [reply doc/Test_Instructions.md:46]: Ok, still not convinced for proto. Why this is needed while creating a FS ? How the proto value is related to FS creation ?

Comment (pratyush-seagate) [reply doc/Test_Instructions.md:46]: Yes you are right. FS creation should not be dependent on protocol usage. It is currently a limitation in the validation code of cortxfscli which needs to be updated to remove the use of proto during FS create.

Comment (sachinpunadikar) [reply doc/Test_Instructions.md:46]: ok, lets have a defect opened for this

**Review by sachinpunadikar — APPROVED** [review]
+1

File: doc/Test_Instructions.md (hunk 1 above)
Comment (pratyush-seagate) [reply doc/Test_Instructions.md:46]: Thanks, https://jts.seagate.com/browse/EOS-15477 is created to track this change.

**Review by s-arya — APPROVED** [review]
LGTM,  +1

**Pull request merged**